docs_src/design/legacy-requirements/README.md

```diff
@@ -1 +1,8 @@
 # Legacy Requirements
+
+| Name/Link                                          | Short Description                            |
+| -------------------------------------------------- | -------------------------------------------- |
+| [Device Service](device-service.md)                | Device Service SDK required functionality    |
```

Comment:
This should be the TOC.md and each doc have a leading 3 digit number?
Same as this: https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/design/TOC.md

Reply:
I was taking the lead from https://github.com/edgexfoundry/edgex-docs/blob/master/docs_src/design/legacy-design/README.md